pkg/signing/signing.go

```diff
@@ -315,12 +328,13 @@ func Verify(cpath, url string, id uint32, isGroup bool, authToken string, noProm
 		signer, err := openpgp.CheckDetachedSignature(elist, bytes.NewBuffer(block.Bytes), block.ArmoredSignature.Body)
 		if err != nil {
 			// verification with local keyring failed, try to fetch from key server
-			sylog.Infof("key missing, searching key server for KeyID: %s...", fingerprint[24:])
-			netlist, err := sypgp.FetchPubkey(fingerprint, url, authToken, noPrompt)
+			sylog.Infof("Image '%v' is signed with key(s) that are not in your keyring.", cpath)
+			sylog.Infof("Searching 'https://keys.sylabs.io' for key (0x%X)...", fingerprint[24:])
```

Comment:
fingerprint[24:] is showing up in multiple places. It's probably a good idea to have a keyID function that extracts the relevant part of the fingerprint.

Reply:
I'm actually going to try to un-change this function `Verify()`, then do all the changes in #2821, To minimize the conflict.

Reply:
Ok